File: LV2/JadenCase 문자열 만들기/solution.py
def solution(s):
    answer = ''
    arr = s.split(' ');
    newArr = []
    for word in arr : 
        word = word.lower()
        word = word.capitalize()
        print(word)
        newArr.append(word)
    
    answer = ' '.join(newArr)
    return answer

#https://school.programmers.co.kr/learn/courses/30/lessons/12951?language=python3

File: LV2/JadenCase 문자열 만들기/solution.js
function solution(s) {
    var answer = '';
    var arr = s.split(' ');
    
    for(var i = 0; i<arr.length; i++){
        var a= arr[i].charAt(0).toUpperCase();
        var b = arr[i].slice(1).toLowerCase();
        answer+= a+b;
        if(i!=arr.length-1) answer+=' '
    }
    return answer;
}

//https://school.programmers.co.kr/learn/courses/30/lessons/12951